documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1718,7 +1718,7 @@ This means that values that are larger than around 8 KBs need to be stored by us
 This impacts replication messages that are coming from the database. Values that were stored by using the TOAST mechanism and that have not been changed are not included in the message, unless they are part of the table's replica identity.
 There is no safe way for {prodname} to read the missing value out-of-bands directly from the database, as this would potentially lead to race conditions. Consequently, {prodname} follows these rules to handle toasted values:
 
-* Tables with `REPLICA IDENTITY FULL` - TOAST column values are part of the `before` and `after` fields in change events just like any other column.
+* Tables with `REPLICA IDENTITY FULL` - unchanged TOAST column values are part of the `before` block, so connector is able to ingest them as if they were provided in `after` block.
```

Comment:
So I'm not quite following on this one; because changed TOAST column values will be part of the before and after blocks in any case, right? So I'd say the sentence was right before; and while the proposed new one also is right, it describes only a subset of the entire situation and may imply to the reader that the same wouldn't be the case for changed values. Perhaps I'm missing something?

Reply:
Honestly, I am not sure we need this change. As I wrote [here](https://issues.redhat.com/browse/DBZ-3835?focusedCommentId=18860970&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-18860970) _it may even be superfluous as it pushes additional unnecessary cognitive pressure_.
So, I'll better close this PR.